README.md

```diff
@@ -45,18 +45,20 @@ The Parse Android SDK has the following Android API and [Gradle Plugin][gradle-p
 
 ## Add Dependency
 
-Add this in your root `build.gradle` file (**not** your module `build.gradle` file):
```

Comment:
Should we mentioned both approaches and add a note to which Android Studio versions they apply?

Reply:
Yeah, was on the fence about it too. I'll add it.

Reply:
The new commit has instructions included again for older versions in a dropdown field. I'm guessing this info isn't needed for most users at this point, and increasingly fewer as time goes on, I chose to nest it inside a drop-down to keep it out of the way for standard setup procedures. But is still accessible for those needing it.